Modules/Forum/classes/class.ilForum.php

```diff
@@ -1792,16 +1792,6 @@ public function mergeThreads(int $source_id, int $target_id) : void
             $targetThreadForMerge->reopen();
         }
 
-        $this->event->raise(
```

Comment:
This `case` MUST NOT be deleted IMO. We need to refresh the learning progress if two threads are combined into one thread.